seirmo/SEIRModelClass.py

```diff
@@ -0,0 +1,42 @@
+"""#
```

Comment:
Could you please delete this file. It is saved in the branch issue-06-SEIR-model, so we can merge the github actions.